.buildkite/scripts/steps/integration-package.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+source .buildkite/scripts/common.sh
+
+AGENT_PACKAGE_VERSION="$(cat .package-version)" PACKAGES=tar.gz,zip,rpm,deb PLATFORMS=linux/amd64,linux/arm64,windows/amd64  SNAPSHOT=true EXTERNAL=true  DEV=true mage package
```

Comment:
To take into account the changes in #5119 this line should be 
```suggestion
PACKAGES=tar.gz,zip,rpm,deb PLATFORMS=linux/amd64,linux/arm64,windows/amd64  SNAPSHOT=true EXTERNAL=true  DEV=true mage package
```